careers/junior-mid-senior.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Junior/Mid/Senior positioning
+description: How we define your role in the team
+---
+
+### Recently we have published a great e-book about how we position our team members regarding their seniority.
+
+## The basic idea is there are 3 levels per tier:
+
+-   Junior 1
+-   Junior 2
+-   Junior 3
+-   Mid 1
+-   Mid 2
+-   Mid 3
+-   Senior 1
+-   Senior 2
+-   Senior 3
+
+## In addition, a possible add-on modifier to your position is:
+
+-   Team Lead 2
+-   Team Lead 1
+-   Team Lead 3
+
+Which define your role in developing people and making them grow on the seniority scale.
```

Comment:
This add-on defines your role as team leader responsible for team members' career development and their position.

Reply:
nice